### ISSUE TITLE
'TS_USER_AGENT' was not declared in this scope

### ISSUE THREAD

**Issue by mathresearch** [mathworks/thingspeak-particle#2]
Veryfing code using this library in Particle Build fails with this error:

```
ThingSpeak/ThingSpeak.h:1369:28: error: 'TS_USER_AGENT' was not declared in this scope
   if (!this->client->print(TS_USER_AGENT)) return false
```

**Comment by rpurser47** [comment]
Hi,

Sorry for the delay in getting back to you.  Which Particle device are you using?   Are you working in the http://build.particle.io web IDE?

-Rob

**Comment by rpurser47** [comment]
Accidentally closed.

**Comment by mathresearch** [comment]
Hi,
Now I have to apologize for replying that late. I was using Particle's web IDE and when trying to compile a simple program I got the error message above.
Bernd

**Comment by briandg** [comment]
Hello,

Just got the same issue as well.  Trying to use the code on a Particle Code and programming over the Particle web IDE.  I did not flash the application, just tried to Verify it first and got this error.

**Comment by jasontwinters** [comment]
@briandg    Because the compiler can't find TS_USER_AGENT, I suspect it might be related to a particular device.  Can you tell me which device you are using?

Also, does this error pop up when attempting to verify the "CheerLight.ino" example?

**Comment by briandg** [comment]
My apologies.  I didn't realize I was trying to upload it to a Photon Redboard, which is known as a P1, which is apparently incompatible with the Thingspeak Library.  I tried it on my of my actual Particle Photons and I was able to verify it (example and my code).  I also was able to verify it on my Particle Core which is the device I wish to use.  I currently have no issues.  Thank you for your reply.

Is there a way for me to retract my declaring this an issue or should I leave these comments here?
